blocks/liveby-map/liveby-map-delayed.js

```diff
@@ -0,0 +1,80 @@
+/* global google */
+
+import { fetchPlaceholders } from '../../scripts/lib-franklin.js';
+
+function getCenter(coords) {
+  // Find bounding box
+  const minX = coords.reduce((x1, x2) => Math.min(x1, x2[0]), 10000);
+  const maxX = coords.reduce((x1, x2) => Math.max(x1, x2[0]), -10000);
+  const minY = coords.reduce((y1, y2) => Math.min(y1, y2[1]), 10000);
+  const maxY = coords.reduce((y1, y2) => Math.max(y1, y2[1]), -10000);
+  // Return center of bounding box
+  return [(maxX + minX) / 2, (maxY + minY) / 2];
+}
+
+function convertCoordinates(coords) {
+  return coords.map((c) => ({
+    lng: parseFloat(c[0]),
+    lat: parseFloat(c[1]),
+  }));
+}
+
+function initLiveByMap() {
+  const mapDiv = document.querySelector('.liveby-map-main');
+  const coordinates = window.liveby.geometry.coordinates[0];
+  const mapCenter = getCenter(coordinates);
+  const map = new google.maps.Map(mapDiv, {
+    zoom: 12,
+    maxZoom: 18,
+    center: {
+      lng: parseFloat(mapCenter[0]),
+      lat: parseFloat(mapCenter[1]),
+    },
+    mapTypeId: 'roadmap',
+    clickableIcons: true,
+    gestureHandling: 'greedy',
+    visualRefresh: true,
+    disableDefaultUI: false,
+  });
+
+  const polyOptions = {
+    map,
+    strokeColor: '#BA9BB2',
+    strokeWeight: 2,
+    fillColor: '#BA9BB2',
+    fillOpacity: 0.3,
+    clickable: false,
+    zIndex: 1,
+    path: convertCoordinates(coordinates),
+    editable: false,
+  };
+
+  // eslint-disable-next-line no-unused-vars
+  const poly = new google.maps.Polygon(polyOptions);
```

Comment:
do you need to store this, if you aren't going to edit it later? Or does lint also complain about the ignored return value of the polygon instantiation?

Reply:
Yeah it was exactly a lint problem.  I tried giving it a _ prefix to indicate it was ignored but it still barked at that.  And calling the new constructor was getting a complaint about a side-effect, etc.  Really just not being nice to me yesterday. :D